app/components/Editor/components/Toolbar/Toolbar.js

```diff
@@ -6,7 +6,6 @@ import { Portal } from 'react-portal';
 import { Editor, findDOMNode } from 'slate-react';
 import { Node, Value } from 'slate';
 import styled from 'styled-components';
-import _ from 'lodash';
```

Comment:
random linting errors exposed by updated dependencies or our CI is not working right